core/primitives/src/version.rs

```diff
@@ -92,7 +95,7 @@ pub const PROTOCOL_VERSION: ProtocolVersion = 41;
 
```

Comment:
You need to bump regular protocol version above, since it's not a nightly feature.

Reply:
Fixed